internal/backends/mcu/README.md

```diff
@@ -1,9 +1,11 @@
-**NOTE**: This library is an **internal** crate of the [Slint project](https://slint-ui.com).
-This crate should **not be used directly** by applications using Slint.
-You should use the `slint` crate instead.
+> **Note**
```

Comment:
note that we load that file in lib.rs so that the docs of the i-slint-backend-mcu crate shows this.

Obviously we won't have it always like that and we want to be able to use the mcu port without an extra backend.  (and this would be rename somehow)

Reply:
Is this about the extra Github Note markdown?

Reply:
yes, i'm not sure why use qutation mark here.  But that's not important

Reply:
Because that renders nicer with the quotation mark.

See [here](https://github.com/github-community/community/discussions/16925).